Doc/Tutorial/chapter_phylo.tex

```diff
@@ -676,44 +676,38 @@ \section{Running external applications}
 \label{sec:PhyloApps}
 
 While Bio.Phylo doesn't infer trees from alignments itself, there are third-party
-programs available that do. These are supported through the module
-\texttt{Bio.Phylo.Applications}, using the same general framework as
-\texttt{Bio.Emboss.Applications}, \texttt{Bio.Align.Applications} and others.
+programs available that do. These can be accessed from within python by using the
+\verb|subprocess| module.
 
-\emph{WARNING:} We have decided to drop these command line wrappers in a future Biopython
-release. We will be updating this documentation to instead build the command line
-directly, and invoke it with the \texttt{subprocess} module.
-
-Biopython 1.58 introduced a wrapper for PhyML
+Bellow is an example on how to use a python script to interact with PhyML
 (\url{http://www.atgc-montpellier.fr/phyml/}). The program accepts an input alignment in
 \texttt{phylip-relaxed} format (that's Phylip format, but without the 10-character limit
-on taxon names) and a variety of options. A quick example:
+on taxon names) and a variety of options.
 
 \begin{minted}{pycon}
->>> from Bio import Phylo
->>> from Bio.Phylo.Applications import PhymlCommandline
->>> cmd = PhymlCommandline(input="Tests/Phylip/random.phy")
->>> out_log, err_log = cmd()
+>>> import subprocess
+>>> cmd = "phyml -i Tests/Phylip/random.phy"
+>>> results = subprocess.run(cmd, shell=True, stdout=subprocess.PIPE, text=True)
 \end{minted}
 
+The `stdout = subprocess.PIPE` argument makes the output of the program accessible through
+`results.stdout` for debugging purposes, (the same can be done for `stderr`), and
+`text=True` makes the returned information be a python string, intead of a `bytes` object.
+
 This generates a tree file and a stats file with the names
-[\textit{input~filename}]\verb|_phyml_tree.txt| and
-[\textit{input~filename}]\verb|_phyml_stats.txt|. The tree file is in Newick format:
+\textit{input~filename}]\verb|_phyml_tree.txt| and
+[\textit{input~filename}]\verb|_phyml_stats.txt|.  The tree file is in Newick format:
```

Comment:
Looks like you lost the opening square bracket (round the italics *input filename*). And added a white space but that's harmless.

Reply:
Fixed it, thanks for catching it.